components/layout/EntryLayout.tsx

```diff
@@ -120,7 +125,7 @@ export const EntryLayout = (props: EntryLayoutProps) => {
       <EntryDetails {...entry} />
       {entry?.terms?.length && <Chart
         rollupToPatternClass={showRollup}
-        showLabels={true}
+        showLabels={window?.innerWidth > 450 ? true : false}
```

Comment:
since the name of the bar is in the expandable section, i think we can get away with hiding chart labels altogether on tiny screens - will look even better once unique icons per row too i suspect!

left row height at 10 so icons remain easily readable, but open to suggestions to shorten on mobile

Reply:
to investigate: `window.innerWidth` is available for me locally, but I'm not actually sure it is when the site builds? check Next docs on how to handle this

Reply:
there are very few instances of needing to check window width currently, so i don't think this is a dealbreaker, but making a notion card to look into adding custom hook for this since the published site won't have `window`

this solution looks fine! https://dev.to/adrien/creating-a-custom-react-hook-to-get-the-window-s-dimensions-in-next-js-135k